raiden/transfer/state.py

```diff
@@ -1646,31 +1606,44 @@ def __init__(
 
     def __repr__(self):
         return '<NettingChannelState id:{} opened:{} closed:{} settled:{} updated:{}>'.format(
-            self.identifier,
+            self.canonical_identifier.channel_identifier,
             self.open_transaction,
             self.close_transaction,
             self.settle_transaction,
             self.update_transaction,
         )
 
+    # FIXME: transitional property -- should go away
```

Comment:
How? what could be changed that will facilitate removing these properties?
Could you document what needs to be done here?

Reply:
my argument is, that we should hardly ever handle the sub components of the canonical identifier at all, instead do all unifications/identifications with the `.canonical_identifier` property. IF we happen to end in a place, where we need the subcomponents, we can just use the full path, i.e.
```
myinstance.canonical_identifier.token_network_address
```